core/scripts/screenshot/ionic.js

```diff
@@ -0,0 +1,78 @@
+const ScreenshotConnector = require('@stencil/core/screenshot/local-connector');
```

Comment:
Do these scripts need to be included? Our screenshot tests don't use this at all AFAIK.